_posts/2024-01-17-geospatial-techno.md

```diff
@@ -0,0 +1,82 @@
+---
+author: GeoSpatial Techno
+layout: post
+title: GeoServer About & Status - A Practical Guide
+date: 2024-01-17
+categories:   
+- Tutorials
+---
+
+[GeoSpatial Techno](https://www.youtube.com/@geospatialtechno) is a startup focused on geospatial information that is providing e-learning courses to enhance the knowledge of geospatial information users, students, and other startups. The main approach of this startup is providing quality, valid specialized training in the field of geospatial information.
+
+( [YouTube](https://www.youtube.com/@geospatialtechno)
+| [LinkedIn](https://www.linkedin.com/in/geospatialtechno)
+| [Facebook](https://www.facebook.com/geospatialtechno)
+| [Reddit](https://www.reddit.com/user/geospatialtechno)
+| [X](https://twitter.com/geospatialtechn)
+)
+
+----
+
+### GeoServer About & Status along with practical guides
+In this session, we would like to talk around the "About & Status" section of GeoServer. If you want to access the complete tutorial, simply click on the [link](https://youtu.be/F-RlvlwwJgQ)
+
+[![](https://img.youtube.com/vi/F-RlvlwwJgQ/0.jpg)](https://www.youtube.com/watch?v=F-RlvlwwJgQ)
+
+### Introduction
+The "About & Status" section of GeoServer provides information about runtime variables and how GeoServer is described to clients that connect to it. In other words, this section provides access to GeoServer diagnostic and configuration tools and can be particularly useful for debugging.
+- "Server Status": view server configuration and run-time status.
+- "GeoServer Logs": see GeoServer log output for error diagnosis.
+- "Contact Information": manage public contact details for WMS server.
+- "About GeoServer": access GeoServer docs, homepage, and bug tracker. You do not need to be logged into GeoServer to access this page.
```

Comment:
This text is a summary of the video I guess?

- Suggestion: Do you wish to cover the welcome page here also?
- Suggestion: You could link to the use manual for each page (in case they want to know more)

Reply:
Each of these sections will be explained in the following paragraphs.